File: lab2-2.cpp
#include <iostream>
using namespace std;
int main() {

    int n;
    cout << "Please enter a positive integer between 10 and 100:";
    cin >> n;

    for (int i = 1; i <= n; i++) {
    if (i % 3) {
     cout << "Fizz" << endl;
    } else
        if (i % 5 == 0) {
            cout << "Buzz" << endl;
    } else {
        cout << i << endl;
    }

    }

    return 0;
}

File: lab2-2.py
while True:
 x = int(input("Give a number between 10 and 100:"))
 if x < 10 or x > 100:
    print('Invalid input. Please enter a number between 10 and 100')
    continue
 break

for n in range(1, x + 1, 1):
    if n % 3 == 0:
        print("Fizz")
    elif n % 5 == 0:
        print("Buzz")
    else:
         print(n)


